pkg/crypto/certificatemanager/certificate_manager.go

```diff
@@ -150,19 +182,23 @@ func (m *manager) GetTLSContext(ctx context.Context, tlsCtx *api.TLSContext, ns
 	if ok {
 		private = string(privateBytes)
 	} else if tlsCtx.PrivateKey != "" {
-		return "", "", "", fmt.Errorf("Private Key %s not found in secret %s", privateName, name)
+		return "", "", "", false, fmt.Errorf("Private Key %s not found in secret %s", privateName, name)
 	}
 
 	if caBytes == nil && publicBytes == nil && privateBytes == nil {
-		return "", "", "", fmt.Errorf("TLS certificates not found in secret %s ", name)
+		return "", "", "", false, fmt.Errorf("TLS certificates not found in secret %s ", name)
 	}
 
-	return ca, public, private, nil
+	// TODO(youngnick): Follow up PR that will change this to a deprecation warning once we actually
+	// mark read-from-file as deprecated.
+	m.Logger.WithField("secret", name).Debug("Secret read from file")
+	return ca, public, private, fromFile, nil
```

Comment:
I've updated the name of the boolean to `inlineSecrets` to make the purpose clearer, but this should actually be `true`, since the Secret is being sent inline via NPDS. Updated.